tests/compile-fail/alloc/reallocate-bad-size.rs

```diff
@@ -5,6 +5,6 @@ use std::alloc::{alloc, realloc, Layout};
 fn main() {
     unsafe {
         let x = alloc(Layout::from_size_align_unchecked(1, 1));
-        realloc(x, Layout::from_size_align_unchecked(2, 1), 1);
+        let _ = realloc(x, Layout::from_size_align_unchecked(2, 1), 1);
```

Comment:
I know `let _ = ...` has different semantics from `let _x = ...` currently (and it seems it's different from `...;` too). Should I change this to `let _x = ` just in case?

Reply:
Semantics are the same when the RHS of `=` is a function call... but yeah, if only for consistency in tests/compile-fail/alloc/reallocate-change-alloc.rs I think it'd be better to use `_y`.

Reply:
Sounds good.